tests/unit_tests/test_db_handler.py

```diff
@@ -0,0 +1,276 @@
+#!/usr/bin/python3
+
+import pytest
+import logging
+# import subprocess
+from pathlib import Path
+
+import simtools.config as cfg
+import simtools.io_handler as io
+from simtools import db_handler
+
+logger = logging.getLogger()
+logger.setLevel(logging.DEBUG)
+
+
+@pytest.fixture
+def db(set_db):
+    db = db_handler.DatabaseHandler()
+    return db
+
+
+@pytest.fixture
+def DB_CTA_SIMULATION_MODEL():
+    return "CTA-Simulation-Model"
+
+
+def test_reading_db_lst(db):
+
+    logger.info("----Testing reading LST-----")
+    pars = db.getModelParameters("north", "lst-1", "Current")
+    if cfg.get("useMongoDB"):
+        assert pars["parabolic_dish"]["Value"] == 1
+        assert pars["camera_pixels"]["Value"] == 1855
+    else:
+        assert pars["parabolic_dish"] == 1
+        assert pars["camera_pixels"] == 1855
+
+    # TODO - is this part of testing the reading from the DB?
+    db.exportModelFiles(pars, io.getTestOutputDirectory())
+    logger.info("Listing files written in {}".format(io.getTestOutputDirectory()))
+    # subprocess.call(["ls -lh {}".format(io.getTestOutputDirectory())], shell=True)
+
+
+def test_reading_db_mst_nc(db):
+
+    logger.info("----Testing reading MST-NectarCam-----")
+    pars = db.getModelParameters("north", "mst-NectarCam-D", "Current")
+    if cfg.get("useMongoDB"):
+        assert pars["camera_pixels"]["Value"] == 1855
+    else:
+        assert pars["camera_pixels"] == 1855
+
+
+def test_reading_db_mst_fc(db):
+
+    logger.info("----Testing reading MST-FlashCam-----")
+    pars = db.getModelParameters("north", "mst-FlashCam-D", "Current")
+    if cfg.get("useMongoDB"):
+        assert pars["camera_pixels"]["Value"] == 1764
+    else:
+        assert pars["camera_pixels"] == 1764
+
+
+def test_reading_db_sst(db):
+
+    logger.info("----Testing reading SST-----")
+    pars = db.getModelParameters("south", "sst-D", "Current")
+    if cfg.get("useMongoDB"):
+        assert pars["camera_pixels"]["Value"] == 2048
+    else:
+        assert pars["camera_pixels"] == 2048
+
+
+def test_copy_telescope_db(db, DB_CTA_SIMULATION_MODEL):
+
+    logger.info("----Testing copying a whole telescope-----")
+    db.copyTelescope(
+        DB_CTA_SIMULATION_MODEL,
+        "North-LST-1",
+        "Current",
+        "North-LST-Test",
+        "sandbox",
+    )
+    db.copyDocuments(
+        DB_CTA_SIMULATION_MODEL,
+        "metadata",
+        {"Entry": "Simulation-Model-Tags"},
+        "sandbox",
+    )
+    pars = db.readMongoDB(
+        "sandbox", "North-LST-Test", "Current", io.getTestOutputDirectory(), False
+    )
+    assert pars["camera_pixels"]["Value"] == 1855
+
+    # TODO - no sure what is tested below
```

Comment:
That we can delete entries. However, maybe we can try to read the entry and have a real `assert` for this test. I will open an issue to remember to check.

Reply:
I guess this is issue #292